services/processor/native/sanitize_code.go

```diff
@@ -27,7 +34,40 @@ func (s *service) createSanitizer() *sanitizer.Sanitizer {
 			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/safemath"`: true,
 			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/service"`:  true,
 			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/state"`:    true,
+
+			// Text
+			`"fmt"`:           true,
```

Comment:
@netoneko At this stage, since we do not support non deterministic execution, I suggest to remove fmt

Reply:
Done.